Signal/src/view controllers/FullImageViewController.m

```diff
@@ -147,6 +147,10 @@ - (void)initializeImageView {
         self.imageView.userInteractionEnabled       = YES;
         self.imageView.clipsToBounds                = YES;
         self.imageView.layer.allowsEdgeAntialiasing = YES;
+        // Use trilinear filters for better scaling quality at
+        // some performance cost.
+        self.imageView.layer.minificationFilter = kCAFilterTrilinear;
+        self.imageView.layer.magnificationFilter = kCAFilterTrilinear;
```

Comment:
Note that this only applies to still images.  Animated gifs are low-res and don't give the eye time to examine fine detail.